Applications/ResNet/res/resnet18.ini

```diff
@@ -0,0 +1,505 @@
+epsilon = 1.0e-6
+momentum = 0.9
+Activation = relu
+beta_initializer = zeros
+gamma_initializer = ones
+moving_mean_initializer = zeros
+moving_variance_initializer = ones
+
+[B3_residue_4_conv_14]
+Type=conv2d
+input_layers = B3_residue_4_bn
+kernel_size = 3,3
+bias_initializer = zeros
+weight_initializer = xavier_uniform
+filters=256
+stride =1,1
+padding=1,1
+
+[B3_2_add]
+Type=addition
+input_layers=B3_residue_4_conv_14, out5
+
+[B3_2_bn]
+Type=batch_normalization
+input_layers = B3_2_add
+epsilon = 1.0e-6
+momentum = 0.9
+Activation = relu
+beta_initializer = zeros
+gamma_initializer = ones
+moving_mean_initializer = zeros
+moving_variance_initializer = ones
+
+[out6]
+Type=output
+input_layers=B3_2_bn
+
+[B4_residue_1_conv_15]
+Type=conv2d
+input_layers = out6
+kernel_size = 3,3
+bias_initializer = zeros
+weight_initializer = xavier_uniform
+filters=512
+stride =2,2
+padding=1,1
+
+[B4_residue_2_bn]
+Type=batch_normalization
+input_layers = B4_residue_1_conv_15
+epsilon = 1.0e-6
+momentum = 0.9
+Activation = relu
+beta_initializer = zeros
+gamma_initializer = ones
+moving_mean_initializer = zeros
+moving_variance_initializer = ones
+
+[B4_residue_2_conv_16]
+Type=conv2d
+input_layers = B4_residue_2_bn
+kernel_size = 3,3
+bias_initializer = zeros
+weight_initializer = xavier_uniform
+filters=512
+stride =1,1
+padding=1,1
+
+[B4_original_reshaped]
+Type=conv2d
+input_layers = out6
+kernel_size = 3,3
+bias_initializer = zeros
+weight_initializer = xavier_uniform
+filters=512
+stride =2,2
+padding=1,1
+
+[B4_1_add]
+Type=addition
+input_layers=B4_residue_2_conv_16, B4_original_reshaped
+
+[B4_1_bn]
+Type=batch_normalization
+input_layers = B4_1_add
+epsilon = 1.0e-6
+momentum = 0.9
+Activation = relu
+beta_initializer = zeros
+gamma_initializer = ones
+moving_mean_initializer = zeros
+moving_variance_initializer = ones
+
+[out7]
+Type=output
+input_layers=B4_1_bn
+
+[B4_residue_3_conv_17]
+Type=conv2d
+input_layers = out7
+kernel_size = 3,3
+bias_initializer = zeros
+weight_initializer = xavier_uniform
+filters=512
+stride =1,1
+padding=1,1
+
+[B4_residue_4_bn]
+Type=batch_normalization
+input_layers = B4_residue_3_conv_17
+epsilon = 1.0e-6
+momentum = 0.9
+Activation = relu
+beta_initializer = zeros
+gamma_initializer = ones
+moving_mean_initializer = zeros
+moving_variance_initializer = ones
+
+[B4_residue_4_conv_18]
+Type=conv2d
+input_layers = B4_residue_4_bn
+kernel_size = 3,3
+bias_initializer = zeros
+weight_initializer = xavier_uniform
+filters=512
+stride =1,1
+padding=1,1
+
+[B4_2_add]
+Type=addition
+input_layers=B4_residue_4_conv_18, out7
+
+[out8]
+Type=output
+input_layers=B4_2_add
+
+[max_pooling2d_1]
+Type=pooling2d
+input_layers = out8
+pool_size = 4,4
+stride =4,4
+padding = 0,0
+pooling = max
+
+[average_pooling2d_1]
+Type=pooling2d
+input_layers = out8
+pool_size = 4,4
+stride =4,4
+padding = 0,0
+pooling = average
+
+[concatenate_1]
+Type=concat
+input_layers=max_pooling2d_1, average_pooling2d_1
+
+[linear]
```

Comment:
lets either rename to `conv**` or change type to `fully_connected`. It's the same essentially, but just for readability.

Reply:
yes. it is. but I think it can be as it is to compare the reference.

Reply:
Yes, of course, this naming is just for our convenience.